packages/core/src/conventional-commits/__tests__/get-commits-since-last-release.spec.ts

```diff
@@ -129,5 +129,21 @@ describe('getOldestCommitSinceLastTag', () => {
       );
       expect(result).toEqual({ commitDate: '2022-07-01T00:01:02-04:00', commitHash: 'deadbeef' });
     });
+
+    it('should expect a commit date and hash when using different time zone', async () => {
+      const isIndependent = true;
+      const result = await getOldestCommitSinceLastTag(execOpts, isIndependent, false);
+      const execSpy = (execSync as jest.Mock)
+        .mockReturnValueOnce('')
+        .mockReturnValueOnce('"deadbeef 2022-07-01T00:01:02+01:00"');
```

Comment:
@ghiscoding is there a way to force the mocked returned value to "deadbeef 2022-07-01T00:01:02+01:00"? I still get the value "deadbeef  2022-07-01T00:01:02-04:00" passed.

Reply:
ah yeah I noticed that when I was writing these tests, I inadvertently used the same mocked result across all unit tests which isn`t correct, I just pushed an updated version of the tests just now, can you cherry pick my last commit into your branch and then modify it to your need. Thanks again

```sj
git cherry pick f618ae9
```

Reply:
That worked, Thank you a lot.